src/views/v2/events-bar/form/submit.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * View: Events Bar Form Submit Input
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/events-bar/form/submit.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<input
```

Comment:
Let's use `<button type="submit">` here instead. Also no need for `aria-label`, `name`, or `value`.

Reply:
Any reason in particular to go for `button` instead of `input`?

Reply:
Only in that buttons allow for a wider range of rendering possibilities, input is only limited to text (whatever is defined in the value attr).